x/ibc/testing/transfer.go

```diff
@@ -0,0 +1,11 @@
+package testing
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func NewTransferCoins(dst TestChannel, denom string, amount int64) sdk.Coins {
```

Comment:
I added this before realizing this was added in `ibc-transfer/types`, since I believe Amount will go from `sdk.Coins` to `sdk.Coin` soon, this can be replaced with `GetPrefixedCoin` when that change occurs